net-utils/src/ip_echo_server.rs

```diff
@@ -173,7 +173,11 @@ pub fn ip_echo_server(
 ) -> IpEchoServer {
     tcp_listener.set_nonblocking(true).unwrap();
 
-    let runtime = Runtime::new().expect("Failed to create Runtime");
+    let runtime = tokio::runtime::Builder::new_multi_thread()
+        .thread_name("solIpEchoSrvrRt")
+        .enable_all()
```

Comment:
I would imagine this enabled more features than the default configuration. Do we know what that delta looks like and if there's any meaningful performance/resource difference?

Reply:
Nope, no difference. I included a note about this in the PR description, but marking up the diff would have been more visible. In any case, `Runtime::new()` calls `Builder::new_multi_thread().enable_all()` under the hood. So, these are equivalent:
https://github.com/tokio-rs/tokio/blob/1f924f95f12c362f5116c560acfa005149f110a1/tokio/src/runtime/runtime.rs#L181-L183